src/NodaTime/Duration.cs

```diff
@@ -1190,6 +1190,25 @@ void IXmlSerializable.WriteXml([NotNull] XmlWriter writer)
         [Pure]
         internal decimal ToDecimalNanoseconds() => IsInt64Representable ? ToInt64NanosecondsUnchecked() : ((decimal)days) * NanosecondsPerDay + nanoOfDay;
 
+        /// <summary>
+        /// Returns the larger duration of the given two.
```

Comment:
Righto - will leave that terminology, but add a remark to each method to make it clear.